### PR TITLE
fixed 404 error diplaying

### PR DESCRIPTION
**Pull request by zori7**

### CODE REVIEW

File: docusaurus/src/components/CodeFromFile/CodeFromFile.tsx [hunk 1]
```diff
@@ -37,15 +40,35 @@ const CodeFromFile = ({
   }, [siteConfig, filePath]);
 
   useEffect(() => {
-    fetch(url)
-      .then(response => response.text())
+    const controller = new AbortController();
```
Comment (bmeeder22) [inline docusaurus/src/components/CodeFromFile/CodeFromFile.tsx:43]: This is going to conflict with the other PR.

File: docusaurus/src/components/CodeFromFile/CodeFromFile.tsx [hunk 2]
```diff
@@ -37,15 +40,35 @@ const CodeFromFile = ({
...
+
+    setError(false);
+
+    fetch(url, {
+      signal: controller.signal,
+      headers: {
+        Accept: mime.getType(languageExt) || 'text/javascript',
```
Comment (bmeeder22) [inline docusaurus/src/components/CodeFromFile/CodeFromFile.tsx:50]: What's the purpose of this? is it needed?

**Review by bmeeder22 — COMMENTED** [review]
Couple comments

File: docusaurus/src/components/CodeFromFile/CodeFromFile.tsx (hunk 1 above)
Comment (zori7) [reply docusaurus/src/components/CodeFromFile/CodeFromFile.tsx:43]: Resolved conflicts

File: docusaurus/src/components/CodeFromFile/CodeFromFile.tsx (hunk 2 above)
Comment (zori7) [reply docusaurus/src/components/CodeFromFile/CodeFromFile.tsx:50]: Without this header the request is not failing on not existing files and I can't track the error

**Pull request merged**